src/SourceBuild/content/repo-projects/runtime.proj

```diff
@@ -32,6 +32,8 @@
 
   <PropertyGroup Condition="'$(DotNetBuildSourceOnly)' == 'true'">
     <BuildArgs>$(BuildArgs) /p:UsingToolMicrosoftNetCompilers=false</BuildArgs>
+    <!-- temporary workaround for https://github.com/dotnet/source-build/issues/4728 -->
+    <BuildArgs>$(BuildArgs) /p:InstallerRuntimeIdentifier=linux-$(TargetArchitecture)</BuildArgs>
```

Comment:
Perhaps it needs the plural one:
```suggestion
    <BuildArgs>$(BuildArgs) /p:InstallerRuntimeIdentifiers=linux-$(TargetArchitecture)</BuildArgs>
```

Reply:
No, Viktor just told me that this is due to the arg not getting forwarded to the inner build, I'll push a fix

Reply:
It was being forwarded: https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_apis/build/builds/866356/logs/17

> 2024-11-11T12:42:32.4300712Z /vmr/repo-projects/Directory.Build.targets(477,5): error MSB3073: The command "/vmr/src/runtime/build.sh --restore --build --pack --publish --ci --configuration Release -bl /p:DotNetBuildRepo=true /p:DotNetBuildOrchestrator=true /p:RestoreConfigFile=/vmr/artifacts/obj/runtime/NuGet.config /p:SourceBuiltAssetsDir=/vmr/artifacts/assets/Release/ /p:SourceBuiltAssetManifestsDir=/vmr/artifacts/manifests/runtime/ /p:OfficialBuildId=20241110.1 /p:SourceBuiltSymbolsDir=/vmr/artifacts/obj/Symbols/runtime/ /p:DotNetBuildSourceOnly=true /p:PreviouslySourceBuiltNupkgCacheDir="/vmr/prereqs/packages/previously-source-built/" /p:ReferencePackageNupkgCacheDir="/vmr/prereqs/packages/reference/" --arch x64 --os linux /p:TargetRid=centos.9-x64 /p:PortableBuild=false /p:RuntimeOS=linux /p:BaseOS=linux-x64 /p:NetCoreAppToolCurrentVersion=10.0 /p:UsingToolMicrosoftNetCompilers=false /p:InstallerRuntimeIdentifier=linux-x64 /p:SourceBuiltShippingPackagesDir=/vmr/artifacts/packages/Release/Shipping/runtime/ /p:SourceBuiltNonShippingPackagesDir=/vmr/artifacts/packages/Release/NonShipping/runtime/ > /vmr/artifacts/log/Release/runtime/runtime.log 2>&1" exited with code 1. [/vmr/repo-projects/runtime.proj]

Reply:
From what I can see that's the outer repo build, not the inner one that needs the arg.